cpuid.go

```diff
@@ -554,6 +559,54 @@ func maxFunctionID() uint32 {
 	return a
 }
 
+func isHypervisorActive() bool {
+	_, _, i, _ := cpuid(0x1)
```

Comment:
The code above should be inserted, to check if the function is available on the CPU. I believe if you insert this, it should work.